File: Java/Lesson_Eight/Dog.java

/**
 * Write a description of class Dog here.
 * 
 * @author (your name) 
 * @version (a version number or a date)
 */
public class Dog extends Animal
{
    String breed;
    int dogAge;    //this should be their age in Dog years
    
    
    public Dog(){
        
    }
    public int getDogAge(){
        return dogAge;
    }
    public void setDogAge(int age){
        dogAge = age * 7;
    }
    public String getBreed(){
        return breed;
    }
    public void setBreed(String type){
        breed = type;
    }
}

File: Python/Lesson-Eight/Dog.py
from Animal import Animal
class Dog(Animal):

	def __init__(self, name, breed):
		Animal.__init__(self, name, "dog")
		self.name = name
		self.breed = breed

	def getBreed(self):
		return self.breed

	def setName(self, name):
		self.name = name

	def setBreed(self, breed):
		self.breed = breed
